src/urh/dev/native/lib/cbladerf.pxd

```diff
@@ -95,11 +95,12 @@ cdef extern from "libbladeRF.h":
         int bladerf_get_bandwidth(bladerf *dev, bladerf_module module, unsigned int *bandwidth)
 
     IF BLADERF_API_VERSION >= 2:
+      int bladerf_set_bias_tee(bladerf *dev, bladerf_channel ch, bool enable)
+      int bladerf_get_bias_tee(bladerf *dev, bladerf_channel ch, bool *enable)
```

Comment:
Please correct the indent here, so it lines up with the IF blocks above and below.